frontend/public/components/storage/create-pvc.tsx

```diff
@@ -67,9 +67,9 @@ export const CreatePVCForm: React.FC<CreatePVCFormProps> = (props) => {
     },
   ];
   const dropdownUnits = {
-    Mi: 'Mi',
-    Gi: 'Gi',
-    Ti: 'Ti',
+    Mi: 'MiB',
+    Gi: 'GiB',
+    Ti: 'TiB',
```

Comment:
Missed this on last review, but if we update this dropdown, we should also update expand pvc.

https://github.com/openshift/console/blob/master/frontend/public/components/modals/expand-pvc-modal.jsx#L45-L49

Reply:
Fixed the dropdown there too 👍